jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/pinning/ConnectionPinningManager.kt

```diff
@@ -146,23 +146,21 @@ class DefaultConnectionPinningManager(private val project: Project) :
             })
             .icon(AllIcons.General.QuestionDialog)
             .help(HelpIds.EXPLORER_CREDS_HELP.id)
-            .ask(project).apply {
+            .ask(null as Project?).apply {
                 if (this) {
-                    UiTelemetry.click(project, "connection_multiple_auths_yes")
+                    UiTelemetry.click(null as Project?, "connection_multiple_auths_yes")
                 } else {
-                    UiTelemetry.click(project, "connection_multiple_auths_no")
+                    UiTelemetry.click(null as Project?, "connection_multiple_auths_no")
```

Comment:
prefer to pass as a named argument: `(project = null)` instead of casting as Project?